github/acls/ansible-collections/community.vmware.config

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
This should be github/acls/ansible-collections/vmware.config

Reply:
oh, makes sense. Thanks.